dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -33,19 +30,24 @@ class CoursesControllerTest < ActionController::TestCase
     File.stubs(:write)
   end
 
-  teardown do
-    Script.clear_cache
-  end
+  class CoursesQueryCountTests < ActionController::TestCase
+    setup do
+      Script.stubs(:should_cache?).returns true
+      Script.clear_cache
 
-  # Tests for index
+      @unit_group_regular = create :unit_group, name: 'non-plc-course', published_state: SharedConstants::PUBLISHED_STATE.beta
+    end
 
-  test_user_gets_response_for :index, response: :success, user: :teacher, queries: 4
+    test_user_gets_response_for :index, response: :success, user: :teacher, queries: 4
 
-  test_user_gets_response_for :index, response: :success, user: :admin, queries: 4
+    test_user_gets_response_for :index, response: :success, user: :admin, queries: 4
 
-  test_user_gets_response_for :index, response: :success, user: :user, queries: 4
+    test_user_gets_response_for :index, response: :success, user: :user, queries: 4
 
-  # Tests for show
+    test_user_gets_response_for :show, response: :success, user: :teacher, params: -> {{course_name: @unit_group_regular.name}}, queries: 15
```

Comment:
this query count did increase from 8 to 15. I couldn't find an obvious explanation for this, but since this test-only change can't be causing a regression in our production behavior, it seems ok to me to increase this query count without determining a root cause.

Reply:
A bit more investigation reveals that this test case was taking only 8 queries when running the entire suite, but was taking 15 queries when run individually. now it takes the same number of queries whether run individually or as part of the test suite, which seems like an improvement for the developer experience.

Reply:
ooo yes I think this is the test I was talking about that did this. Glad you found the fix for it!

Reply:
excellent! sounds like this was a lucky side effect.